packages/matchbox-site/src/helpers/useComponentProps.js

```diff
@@ -29,7 +29,7 @@ function useComponentProps(name) {
   `);
 
   const nodes = _.get(data, 'allComponentMetadata.nodes');
```

Comment:
Like we discussed last week, I think we should move away from graphql here and manually write out props